samples/rag-cosmosdb-nosql/README.md

```diff
@@ -0,0 +1,62 @@
+# Semantic Search Demo
+
+## Prerequisites
+
+This sample is available in the following language stacks:
+
+- [C# on the isolated worker](csharp-ooproc/)
+- [Python](python/)
+
+Please refer to the [root README](../../README.md#requirements) for common prerequisites that apply to all samples.
```

Comment:
Could you also add a documentation on how to use managed identity settings or key based auth.

Example from Table Storage Auth - https://github.com/Azure/azure-functions-openai-extension/tree/main/samples/chat#chat-storage-configuration

I will test the sample based on these instructions